docs/user/network/dashmate/index.rst

```diff
@@ -412,21 +429,23 @@ Historical files are each limited to 1GB.
 
 Press **Ctrl + X** to close the editor and **Y** and **Enter** save the file.
```

Comment:
Prove a link to log rotate documentation

Reply:
You mean something like: https://manpages.ubuntu.com/manpages/xenial/man8/logrotate.8.html?